website/src/css/customTheme.scss

```diff
@@ -585,14 +585,15 @@ article header h1 {
   line-height: 3rem;
 }
 
-article header h2, {
+article header h2 {
```

Comment:
nit: these changes should be reverted. They are unrelated to the changes we want to push here! :wink:

Reply:
@cipolleschi, I added the stylesheet changes to a separate commit.
Someone should do them at some stage since prettier hasn't been run.
Would you like for me to add them to a separate PR?

Reply:
yeah, that would be great. Also, prettier is running in CI, theoretically... 🤔